rumqttc/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the `Key` enum: users do not need to specify the TLS key variant in the `TlsConfiguration` anymore, this is inferred automatically.
 To update your code simply remove `Key::ECC()` or `Key::RSA()` from the initialization.
 - certificate for client authentication is now optional while using native-tls. `der` & `password` fields are replaced by `client_auth`.
+- Make v5 `RetainForwardRule` public, in order to allow setting it when constructing `Filter` values. 
```

Comment:
```suggestion
- `RetainForwardRule` in v5 is now public
```

let's omit the construction of `Filter` values for now as that might be confusing due to `Filters` aren't yet exposed. ( note: not that necessary, feel free to discuss this suggestion haha )

Reply:
Huh? `Filter` is exposed, and the reason I am making this PR.

Reply:
My bad 😬 I thought the PR of setting v5 subscription options was making it public 😅